test/Models/RegionProviderSpec.js

```diff
@@ -26,9 +25,8 @@ beforeEach(function() {
 
 describe('RegionProvider', function() {
   it('parses WFS xml correctly', function(done) {
-    loadText('test/csv/mini_ced.xml').then(function(xml) {
-        ced.loadRegionsFromXML(xml);
-        expect(ced.regions.length).toEqual(6);
+    ced.loadRegionIDs().then(function(json) {
```

Comment:
This isn't really a comment on your commit, but on the existing code - can you set it up so the unit tests didn't go off to a server at all, eg. using a spy? (eg. see [ArcGisMapServerCatalogItemSpec](https://github.com/TerriaJS/terriajs/blob/master/test/Models/ArcGisMapServerCatalogItemSpec.js#L24))